.claude/skills/observability-instrumentation/SKILL.md

```
@@ -1,593 +1,595 @@
-
-final readonly class CreateCustomerCommandHandler
+// ❌ BAD: Metrics in command handler (violates SRP)
+final class CreateCustomerHandler
 {
     public function __construct(
-        private LoggerInterface $logger,
-        private CustomerRepository $repository
+        private CustomerRepository $repository,
+        private BusinessMetricsEmitterInterface $metrics  // Wrong!
     ) {}
 }
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)